moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp

```diff
@@ -151,7 +151,6 @@ bool PointCloudOctomapUpdater::getShapeTransform(ShapeHandle h, Eigen::Affine3d&
   ShapeTransformCache::const_iterator it = transform_cache_.find(h);
   if (it == transform_cache_.end())
   {
-    ROS_ERROR("Internal error. Shape filter handle %u not found", h);
```

Comment:
Why do you drop this error msg? `getShapeTransform()` could be called in a different context (other than as a callback to `maskContainment()` as well, couldn't it?
I think, you should simply improve the message here, e.g. stating "Missing transform for shape with handle ...".